docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,463 @@
+
+It may take up to 30 seconds to create the `KeptnApp` so run the following command until you see the `keptnappdemo` CRD.
+
+```shell
+kubectl -n keptndemo get keptnapp
+```
+
+Expected output:
+
+```shell
+NAME           AGE
+keptndemoapp   2s
+```
+
+Keptn also creates a new application version every time you increment the `version` label.
+
+The `PHASE` will change as the deployment progresses.
+A successful deployment is shown as `PHASE=Completed`
+
+```shell
+kubectl -n keptndemo get keptnappversion
+```
+
+Expected output:
+
+```shell
+NAME                      APPNAME        VERSION   PHASE
+keptndemoapp-0.0.1-***    keptndemoapp   0.0.1     Completed
+```
+
+Keptn can run tasks and SLO evaluations before and after deployment.
+You haven't configured this yet, but you can see the full lifecycle for a `keptnappversion` by running:
+
+```shell
+kubectl -n keptndemo get keptnappversion -o wide
+```
+
+Keptn applications are a collection of workloads.
+By default, Keptn will build `KeptnApp` CRDs based on the labels you provide.
+
+In the example above, the `KeptnApp` called `keptndemoapp` contains one workload (based on the `name` label):
+
+## View your application
+
+Port-forward to expose your app on `http://localhost:8080`:
+
+```shell
+kubectl -n keptndemo port-forward svc/nginx 8080
+```
+
+You should see the "Welcome to nginx" page.
+
+![nginx demo app](/docs/install/assets/nginx.png)
+
+## View DORA Metrics
+
+Keptn is generating DORA metrics and OpenTelemetry traces for your deployments.
+
+These metrics are exposed via the Keptn lifecycle operator `/metrics` endpoint on port `2222`.
+
+To see these raw metrics, port-forward to the lifecycle operator metrics service:
+
+```shell
+kubectl -n keptn-lifecycle-toolkit-system port-forward service/keptn-klt-lifecycle-operator-metrics-service 2222
+```
+
+Access metrics in Prometheus format on `http://localhost:2222/metrics`.
+Look for metrics starting with `keptn_`.
+
+![keptn prometheus metrics](/docs/install/assets/keptnprommetrics.png)
+
+## Make DORA metrics more user friendly
+
+It is much more user friendly to provide dashboards for metrics, logs and traces.
+So let's install new Observability components to help us:
+
+- Cert manager: Jaeger requires cert-manager
+- Jaeger: Store and view DORA deployment traces
+- Prometheus: Store DORA metrics
+- OpenTelemetry collector: Scrape metrics from the above DORA metrics endpoint & forward to Prometheus
+- Grafana (and some prebuilt dashboards): Visualise the data
+
+![add observability](/docs/install/assets/install01.png)
+
+## Install Cert Manager
+
+Jaeger requires Cert Manager, so install it now:
+
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.2/cert-manager.crds.yaml
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager --namespace cert-manager --version v1.12.2 jetstack/cert-manager --create-namespace --wait
+```
```

Comment:
```suggestion
```
```
Is this the KLT cert-manager?  I thought that was now installed as part of the KLT installation.  Maybe only if you install from the manifest, which we did not do?  Need to clarify this with the core-engineering team.

Reply:
No it isn't. It is the "public" [default cert manager](https://cert-manager.io/). This is the root of my issue with "bundling" a custom cert manager. (The public / standard) cert manager is recommended for and indeed required by most tooling that communicates via TLS these days. So any realistic cluster configuration will end up with either:

1. KLT + it's cert manager
2. The default cert manager for everything else

OR

1. KLT w/o it's cert manager
2. The default cert manager for everything (inc. KLT)

Bundling a cert-manager with KLT seems to be a design decision taken solely to make demos easier.